.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ defaults: &defaults
   working_directory: ~/picassojs
   # Available images https://hub.docker.com/r/circleci/node/tags/
   docker:
-    - image: cimg/node:lts-browsers
+    - image: cimg/node:current-browsers
```

Comment:
Using latest node version to get around this issue https://github.com/nodejs/corepack/issues/627